.github/workflows/release-airgap-artifacts.yaml

```diff
@@ -0,0 +1,51 @@
+name: Release airgap artifacts
+on:
+  push:
+    tags:
+      - 'v*'
+  workflow_dispatch:
+permissions:
+  contents: read
+  id-token: write
+
+jobs:
+  release-airgap-artifacts:
+    name: Start airgap release
+    runs-on:
+      - self-hosted
+      - medium
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - uses: webfactory/ssh-agent@v0.7.0
+        with:
+          ssh-private-key: ${{ secrets.PRIVATE_SSH_KEY }}
+
+      - name: Extract branch name
+        shell: bash
+        run: echo "branch=${GITHUB_HEAD_REF:-${GITHUB_REF#refs/heads/}}" >> $GITHUB_OUTPUT
+        id: extract_branch
+
+      - name: Extract tag name
```

Comment:
we can remove the `Extract branch name` step now and use the `RELEASE_VERSION` env var in the steps where we used the branch output

i.e.
```
CATALOG_APPLICATIONS_VERSION=${{ env.RELEASE_VERSION }}
```